### PR TITLE
Trail counters publishing

### PR DESCRIPTION
**Pull request by Charlie-Henry**
[Link to Socrata dataset (private)](https://data.austintexas.gov/dataset/Trail-Counters-Daily-Totals/26tt-cp67)

Script that goes to the EcoCountersAPI and then puts the data in Socrata!

Next step is Prefect

### CODE REVIEW

**Comment by Charlie-Henry** [comment]
Updated this to work with Docker and Prefect 🔥

**Comment by johnclary** [comment]
@Charlie-Henry i'm so excited that you've got this nut cracked! i'm going to defer to @mateoclarke and colleagues on the review. i don't think we have a huge sense of urgency here, right?

I do think we should we rename this repo to reflect the narrow purpose of ETL-ing these trail counters. Or, if this repo intends to hold more code related to crash analysis, move this ETL to a separate repo. E.g. `atd-eco-counter-data`. What do y'all think?

**Comment by Charlie-Henry** [comment]
I do think it's a good idea to move this to its own repo @johnclary . There is no urgency here, I was doing this more as a test ETL that I could wrap my head around to dockerize and put together in Prefect. It's [running](https://cloud.prefect.io/flow/48f2319d-b885-4c22-bbab-3187cf63a305) in test in Prefect with [this flow](https://github.com/cityofaustin/atd-prefect/blob/trail-counters-flow/flows/trail-counters/trail_counter_pub.py).

File: dockerfile
Comment (frankhereford) [inline dockerfile]: TIL that there is a set of naming conventions in place on `hub.docker.com` for Dockerfiles with relation to their autobuild service. We build our own images, so this isn't a big deal to us, but I think it may be worth `git mv`ing this file to change its name to Dockerfile. I generally see it capitalized FWIW. Not a big deal by any means though.

Comment (frankhereford) [inline dockerfile]: You may consider adding a newline at the end of this file too.

File: README.md
Comment (frankhereford) [inline README.md:5]: I agree that this feels out of place in this repo, but I don't have a lot of context for this repo yet since it's new. Maybe since this concerns an airflow dag, you could make a new folder to hold one-offs and spikes in the prefect repo? Making a new repo too is great as an alternative too, if this is going to be an on-going project. I'm curious what more senior team members think, for sure.

File: .dockerignore
Comment (frankhereford) [inline .dockerignore:1]: Maybe consider a new line at the end of file?

File: utils.py
Comment (frankhereford) [inline utils.py]: Maybe add an EOF newline?

**Review by frankhereford — APPROVED** [review]
Looks good to me! I put a few tiny notes down, but nothing to hold up moving forward by any means. Thanks Charlie - I can't wait to learn more prefect related info from you!

File: utils.py
Comment (Charlie-Henry) [reply utils.py]: Thanks for these @frankhereford!

File: README.md
Comment (mateoclarke) [reply README.md:5]: I kinda think we should've put most of the things in this repo in https://github.com/cityofaustin/atd-vz-data if the output is for Vision Zero for mobility safety.

**Comment by mateoclarke** [comment]
👍

File: README.md
Comment (Charlie-Henry) [reply README.md:5]: Trail counter data was mentioned as a data source for the analysis VZ is doing for ped/bike crashes. I'm leaning towards putting this in its own repo or moving this to a folder in this repo called `data` or something?

**Pull request merged**
